craft_providers/bases/almalinux.py

```diff
@@ -83,7 +83,9 @@ def __init__(
         cache_path: Optional[pathlib.Path] = None,
     ) -> None:
         self._cache_path = cache_path
-        self.alias: AlmaLinuxBaseAlias = alias
+
+        # ignore enum subclass (see https://github.com/microsoft/pyright/issues/6750)
+        self.alias: AlmaLinuxBaseAlias = alias  # pyright: ignore
```

Comment:
Ah, pyright...

Where there is no way to get `self.alias: AlmaLinuxBaseAlias = alias # pyright: ignore[reportIncompatibleVariableOverride]` under 80 characters